src/geocoder_control.js

```diff
@@ -7,6 +7,7 @@ var GeocoderControl = L.Control.extend({
     includes: L.Mixin.Events,
 
     options: {
+        proximity: false,
```

Comment:
Hopefully drastically better results though, right? But, yeah, I can see an argument either way, and it's easy to switch later on. @tmcw want to be a tiebreaker?